daml-assistant/src/DAML/Assistant/Command.hs

```diff
@@ -46,13 +46,13 @@ dispatch info = subcommand
 commandParser :: [SdkCommandInfo] -> Parser Command
 commandParser cmds | (hidden, visible) <- partition isHidden cmds = asum
     [ subparser -- visible commands
-        $  builtin "version" "Display SDK version" mempty (pure Version <**> helper)
-        <> builtin "install" "Install SDK version" mempty (Install <$> installParser <**> helper)
-        <> builtin "exec" "Execute command with daml environment." forwardOptions
-            (Exec <$> strArgument (metavar "CMD") <*> many (strArgument (metavar "ARGS")))
+        $  builtin "version" "Display DAML SDK version" mempty (pure Version <**> helper)
+        <> builtin "install" "Install DAML SDK version" mempty (Install <$> installParser <**> helper)
```

Comment:
```suggestion
        <> builtin "install" "Install the specified DAML SDK version, or the latest version if no version is specified" mempty (Install <$> installParser <**> helper)
```

Reply:
`Install DAML SDK version (defaults to latest)` seems to capture the same thing more concisely (given how `--help` shows, concise matters)

Reply:
It does not default to latest (that is the command `daml install latest`, with `daml install` showing all the available targets). I like the description "Install the specified DAML SDK version".